pages/linux/microcom.md

```diff
@@ -0,0 +1,11 @@
+# microcom
+
+> A minimalistic terminal program.
+
+- Open a serial port using the specified baudrate:
+
+`microcom --port {{path/to/serial-port}} --speed {{baudrate}}`
+
+- Establish a telnet connection (rfc2217) to the specified host:
```

Comment:
We can remove the rfc number. I think the word telnet is ubiquitous enough.

Reply:
RFC number is gone.